overlord/snapstate/handlers.go

```diff
@@ -690,10 +690,15 @@ func (m *SnapManager) startSnapServices(t *state.Task, _ *tomb.Tomb) error {
 	if err != nil {
 		return err
 	}
+	svcs := currentInfo.Services()
+	// TODO:
```

Comment:
I fear is better to bite the bullet here given that you need to touch all the tests anyway.